Supertext.Base.Dal.SqlServer.Tests/Utils/SqlResultConverterTest.cs

```diff
@@ -0,0 +1,163 @@
+using System;
+using System.Collections.Generic;
+using System.Text.Json;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Supertext.Base.Dal.SqlServer.Utils;
+
+namespace Supertext.Base.Dal.SqlServer.Tests.Utils;
+
+[TestClass]
+public class SqlResultConverterTest
+{
+    [TestMethod]
+    public void Level1DateToUtc()
+    {
+        var testData = new List<Dictionary<string, object>>() {
+                new () {
+                    {"a", new DateTime()},
+                    {"b", DateTime.UtcNow}
+                }
+            };
+
+        var converter = new SqlResultConverter();
+        converter.InterpretUtcDates(testData);
+
+        Assert.AreEqual(DateTimeKind.Utc, ((DateTime)testData[0]["a"]).Kind);

```

Comment:
We usually use FluentAssertions for asserting. Here it would be like: `((DateTime)testData[0]["a"]).Kind).Should().Be(DateTimeKind.Utc);`